### ISSUE TITLE
TOC error

### ISSUE THREAD

**Issue by lch32111** [brennanbrown/enjoyment-work#18]
Hello. Thank you for sharing your great blog template!

I'm trying to learn how to write posts. So I made one and then upload it.
However, the display was totally different from my localhost.
The thing is that the toc(table of contents) display the whole content instead of its titles or subtitles.

So, here is my post [address](https://lch32111.github.io/posts/Hey-Gravity-Potential-Energy).
and here is markdown file [address](https://github.com/lch32111/lch32111.github.io/blob/main/_posts/2022-09-13-Hey-Gravity-Potential-Energy.md) from my github.

I even did the "toc : false", but it does not work. Can I know how can I solve this problem?

**Comment by brennanbrown** [comment]
Thank you for this issue @lch32111, I will have to look into this.

For now, the simplest suggestion I can give you is to just make `_includes/toc.html` blank.

I'm afraid that after looking at the code, the logic to turn off TOC generation isn't actually there. Big whoops. I will fix soon.

**Comment by lch32111** [comment]
Yes it works now by removing the `_includes/toc.html`. Thank you.
I think it's good to have a feature to turn off the TOC but, I'd also like to have the TOC feature work well.
If you can do this, I will appreciate it. By the way, I don't know why my blog works like this with the toc.
